packages/frontend/src/hooks/useChartColorConfig.tsx

```diff
@@ -91,13 +91,35 @@ export const isGroupedSeries = (series: SeriesLike) => {
     );
 };
 
+export const calculateSeriesLikeIdentifier = (series: SeriesLike) => {
```

Comment:
Extracted from further down below, mostly unchanged (now returns all identifier components as a tuple, instead of just the part we assume to be useful).

Reply:
This makes sense. Some issues I saw def need this.